server/ugRestApi.test.js

```diff
@@ -0,0 +1,107 @@
+jest.mock('./configuration', () => ({ server: {} }))
+
+const log = require('@kth/log')
+const ugRestApi = require('./ugRestApi')
+
+jest.mock('@kth/log')
+log.info = jest.fn()
+log.debug = jest.fn()
+log.error = jest.fn()
+
+const groupData = [
+  {
+    name: 'edu.courses.SF.SF1624.examiner',
+    members: [
+      {
+        email: 'bichel@kth.se',
+        givenName: 'Johannes',
+        kthid: 'u13fzqax',
+        surname: 'Bichel',
+        username: 'bichel',
+      },
+      {
+        email: 'testuge@kth.se',
+        givenName: 'Test UG Gruppeditor',
+        kthid: 'u11fyan6',
+        surname: 'Detectify',
+        username: 'testuge',
+      },
+      {
+        email: 'elenara@kth.se',
+        emailAddress: 'elenara@kth.se',
+        givenName: 'Elena',
+        kthid: 'u1ask19i',
+        phoneHR: '087906682',
+        surname: 'Rakhimova',
+        username: 'elenara',
+      },
+      {
+        email: 'bichel@kth.se',
+        givenName: 'Johannes',
+        kthid: 'u13fzqax',
+        surname: 'Bichel',
+        username: 'bichel',
+      },
+    ],
+  },
+  {
+    name: 'edu.courses.SF.SF1624.1.courseresponsible',
+    members: [
+      {
+        givenName: 'sabahat',
+        firstName: 'Rao Sabahat Ali',
+        lastName: 'Khan',
+        surname: 'Khan',
+        email: 'sabahat@kth.se',
+        username: 'sabahat',
+      },
+      {
+        email: 'madjidi@kth.se',
+        emailAddress: 'madjidi@kth.se',
+        givenName: 'Payam',
+        kthid: 'u1rw2ong',
+        phoneHR: '087906673',
+        surname: 'Madjidi',
+        username: 'madjidi',
+      },
+      {
+        email: 'jeskla@kth.se',
+        emailAddress: 'jeskla@kth.se',
+        givenName: 'Jessica',
+        kthid: 'u16uygcs',
+        phoneHR: '087906164',
+        surname: 'Klarström',
+        username: 'jeskla',
+      },
+      {
+        givenName: 'sabahat',
+        firstName: 'Rao Sabahat Ali',
+        lastName: 'Khan',
+        surname: 'Khan',
+        email: 'sabahat@kth.se',
+        username: 'sabahat',
```

Comment:
We can't use real users id, surnames, phone numbers and emails in the public code because of GDPR, therefore it should be fake data, f.e., testuser1@kkk.kk

Reply:
so it should be fake users, you can create fake data yourself or by using f.e., https://www.npmjs.com/package/@faker-js/faker

Reply:
Added @faker-js and used fake data instead of actual data of users.